tpgtools/main.go

```diff
@@ -562,6 +562,7 @@ func generateProductsFile(fileName string, products []*ProductMetadata) {
 
 var TemplateFunctions = template.FuncMap{
 	"title":                           strings.Title,
+	"containz":                        strings.Contains,
```

Comment:
I'm a little unclear why this seems to be working as-is?

```suggestion
	"contains":                        strings.Contains,
```

Reply:
loool yeah this is not needed